src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Figure extends FigureSupplier {
```

Comment:
```suggestion
public abstract class Figure {
```
Figure does not extend the functionality of FigureSupplier, they have different purpose. Figure - object representing real life figure. FigureSupplier - generating random values and figures based on them.